README.md

```diff
@@ -1,13 +1,23 @@
 # Project Name
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The task was to create digital version of the character guessing board game Guess Who? by focusing mainly on using JavaScript.
```

Comment:
I really enjoyed reading your code Irina! Clean and effective, keep up the great job! 👍🏼  😀  👯‍♀️